### PR TITLE
Blog announcing inaugural EKS to AKS migration post

### PR DESCRIPTION
**Pull request by KenKilty**
Please review and comment with required changes if any.

### CODE REVIEW

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (seanmck) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:13]: "We're excited to announce" always triggers my marketing detector and I really think we want to avoid that vibe in the blog. I'd suggest sticking to more neutral/fact-based language and let the reader decide whether it's exciting. :)

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (seanmck) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:19]: You're spelling out what EKS means here but you already used the acronym above. Suggest moving the full name to the first place you use it in the article.

Comment (seanmck) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:19]: On the point about minimizing changes - it sounds like there's an implicit tradeoff there that you might want to call out.  Is there some way in which the architecture could be even better if you leaned into azure fully? That could be a good thing to mention as it suggests that there's a spectrum of portability to value that the user can move along based on their goals.

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (seanmck) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:33]: This is kind of a dry ending and I don't think the blog post is long enough to really need a recap. Can we close with something more forward looking, like about what additional guides are coming? Maybe encourage people to reach out somehow with other ideas?

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (seanmck) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:15]: Rather than saying that moving a workload from another managed K8s offering can be challenging, I think it would be more accurate to say something like "Even with a portable API like Kubernetes offers, moving between clouds can be challenging". You could also throw in an explanatory metaphor here to make the post a little more informal - eg. "Every cloud has its own unique concepts and behaviors that can seem foreign when you're accustomed to another cloud's approach, not unlike the experience of learning a new language or visiting a new country for the first time. Think of us like your language tutor and local guide to the world of Azure."

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:13]:
```suggestion
We would like to share the first in a new documentation series designed specifically for customers already using Amazon EKS, to help them replicate or migrate their workloads to AKS: [Replicate an AWS event-driven workflow (EDW) workload with KEDA and Karpenter in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview)
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:15]:
```suggestion
Even with Kubernetes’ portable API, moving between clouds can be challenging. Each cloud has its own unique concepts, behaviors, and quirks that will seem unfamiliar when you’re accustomed to another cloud’s approach. This is not unlike the experience of learning a new language or visiting a new country for the first time. This series will be local guide to the world of Azure. The samples in this series begin with infrastructure and code on EKS and end with equivalently functional infrastructure and code on AKS, while explaining the conceptual differences between AWS and Azure throughout.
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:19]:
```suggestion
This new [article](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview) guides you through replicating an EKS event-driven workflow (EDW) workload using [KEDA](https://keda.sh/) and [Karpenter](https://karpenter.sh/) in Azure Kubernetes Service (AKS). In the spirit keeping the workload as portable as possible, this guide prioritizes minimizing the changes needed to get the workload running on Azure.
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:31]:
```suggestion
## Looking forward
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:33]:
```suggestion
Our next article will explore a common EKS workload pattern for protecting an n-tier web/mobile application with a WAF, and the process of replicating this workload into Azure using EKS and other services. We are interested to hear about the migration challenges and scenarios you encounter, so we can continue to enhance the features and documentation for AKS to ease the transition from other clouds to Azure. Please share your ideas and needs using one of the contact methods below.
```

**Review by KenKilty — COMMENTED** [review]
Updated prose with Sean's feedback, please review.

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (chzbrgr71) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:4]: Update this date to something today or earlier. This is used to render the page. Ideally the file name and this date match

**Review by chzbrgr71 — APPROVED** [review]
Looks ready to publish.

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (phillipgibson) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:15]: I would swap "quirks" for characteristics

Comment (phillipgibson) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:15]: Did you mean "This series will be a local guide to the world of Azure."

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (phillipgibson) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:19]: I believe this sentence needs a "of" - In the spirit of keeping the workload as portable as possible

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (russd2357) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:11]: Accommodating customer cloud preference is a driver for some, but not all customers who use multiple clouds.  "Some of these customers" is more accurate start for this sentence.

Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:11]:
```suggestion
Many companies use multiple clouds for their workloads. Some of these companies need to accommodate the cloud preferences of their customers. Kubernetes plays a central role in multi-cloud workloads due to its ability to provide a consistent and portable environment across different cloud providers.
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:19]:
```suggestion
This new [article](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview) guides you through replicating an EKS event-driven workflow (EDW) workload using [KEDA](https://keda.sh/) and [Karpenter](https://karpenter.sh/) in Azure Kubernetes Service (AKS). In the spirit of keeping the workload as portable as possible, this guide prioritizes minimizing the changes needed to get the workload running on Azure.
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:15]:
```suggestion
Even with Kubernetes’ portable API, moving between clouds can be challenging. Each cloud has its own unique concepts, behaviors, and characteristics that will seem unfamiliar when you’re accustomed to another cloud’s approach. This is not unlike the experience of learning a new language or visiting a new country for the first time. This series will be a local guide to the world of Azure. The samples in this series begin with infrastructure and code on EKS and end with equivalently functional infrastructure and code on AKS, while explaining the conceptual differences between AWS and Azure throughout.
```

**Review by KenKilty — COMMENTED** [review]
Updated with additional feedback. Ready for publishing.

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (chzbrgr71) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:4]: This date needs to be some date now or in the past or the post won't render. Maybe change to 2024-08-15

File: .vscode/ltex.hiddenFalsePositives.en-US.txt
Comment (chzbrgr71) [inline .vscode/ltex.hiddenFalsePositives.en-US.txt:1]: You need to remove these from the PR please

File: .vscode/ltex.dictionary.en-US.txt
Comment (chzbrgr71) [inline .vscode/ltex.dictionary.en-US.txt:1]: You need to remove these from the PR

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (KenKilty) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:4]:
```suggestion
date: 2024-08-01
```

**Review by KenKilty — COMMENTED** [review]
Updated date in the metadata

**Review by KenKilty — COMMENTED** [review]
requested changes made

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (pauldotyu) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:4]: Change the date to ensure it is publishes as soon as it is merged.

```suggestion
date: 2024-08-15
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (pauldotyu) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:13]:
```suggestion
We would like to share the first in a new documentation series designed specifically for customers already using Amazon EKS, to help them replicate or migrate their workloads to AKS: [Replicate an AWS event-driven workflow (EDW) workload with KEDA and Karpenter in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/azure/aks/eks-edw-overview)
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (pauldotyu) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:19]:
```suggestion
This new [article](https://learn.microsoft.com/azure/aks/eks-edw-overview) guides you through replicating an EKS event-driven workflow (EDW) workload using [KEDA](https://keda.sh/) and [Karpenter](https://karpenter.sh/) in Azure Kubernetes Service (AKS). In the spirit keeping the workload as portable as possible, this guide prioritizes minimizing the changes needed to get the workload running on Azure.
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (pauldotyu) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:21]:
```suggestion
The sample workload implements the competing consumers pattern with a producer/consumer app, facilitating efficient data processing by separating data production from data consumption. KEDA is used to scale pods running consumer processing, while AKS managed Karpenter autoscale Kubernetes nodes through the new AKS [Node Autoprovisioning (NAP)](https://learn.microsoft.com/azure/aks/node-autoprovision) feature currently available in preview.
```

File: blog/_posts/2024-08-07-eks-to-aks-series-part1.md
Comment (pauldotyu) [inline blog/_posts/2024-08-07-eks-to-aks-series-part1.md:31]:
```suggestion
## Looking ahead
```

**Review by pauldotyu — CHANGES_REQUESTED** [review]
Nice way to introduce a new doc! Some minor copy updates, mostly around removing locales from MS Learn URLs.

**Comment by chzbrgr71** [comment]
Changes look good and we are ready to merge. Need a 2nd approver

**Review by phillipgibson — APPROVED** [review]
LGTM

**Pull request merged**
